pallets/staking/src/lib.rs

```diff
@@ -1315,9 +1317,9 @@ decl_storage! {
         /// forcing into account.
         pub IsCurrentSessionFinal get(fn is_current_session_final): bool = false;
 
-        /// The map from (wannabe) validators to the status of compliance.
-        pub PermissionedValidators get(fn permissioned_validators):
-            map hasher(twox_64_concat) T::AccountId => bool;
+        /// Allowed entities who shows interests to run operators/validator node.
```

Comment:
```suggestion
        /// Allowed entities who shows interests in running operators/validator node.
```

Reply:
I'd rephrase this to something like "Entities that are allowed to run operator/validator nodes".